jbpm-bpmn2/src/main/java/org/jbpm/bpmn2/xml/AbstractNodeHandler.java

```diff
@@ -16,9 +16,6 @@
 
 package org.jbpm.bpmn2.xml;
 
-import static org.jbpm.bpmn2.xml.ProcessHandler.PROCESS_INSTANCE_SIGNAL_EVENT;
-import static org.jbpm.bpmn2.xml.ProcessHandler.RUNTIME_MANAGER_SIGNAL_EVENT;
-import static org.jbpm.bpmn2.xml.ProcessHandler.RUNTIME_SIGNAL_EVENT;
```

Comment:
no, it should still be there as it is used by other types of events in different format. I removed a shared once for signals to make use of different approach, but others still need it.